text/0201-scope-attributes.md

```diff
@@ -0,0 +1,209 @@
+# Introduce Scope Attributes
+
+This OTEP adds attributes to the Scope.
```

Comment:
```suggestion
This OTEP adds attributes to the scope of a telemetry emitter (e.g. Tracer, Meter, Logger).
```

"scope" without further qualification is too ambigous. E.g. I had hoped this might bring a solution for https://github.com/open-telemetry/opentelemetry-specification/issues/335 / https://github.com/open-telemetry/opentelemetry-specification/issues/1143 but this addresses a different problem.

Reply:
Fixed.